e2e/scenario_test.go

```diff
@@ -1187,3 +1187,42 @@ func Test_ubuntu1804imdsrestriction_mangletable(t *testing.T) {
 		},
 	})
 }
+
+func Test_Ubuntu2204MessageOfTheDay(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Tags: Tags{
+			ServerTLSBootstrapping: true,
+		},
+		Description: "tests that a node on ubuntu 2204 bootstrapped and message of the day is properly added to the node",
+		Config: Config{
+			Cluster: ClusterKubenet,
+			VHD:     config.VHDUbuntu2204Gen2Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
+				nbc.AgentPoolProfile.MessageOfTheDay = "Zm9vYmFyDQo=" // base64 for foobar
+			},
+			LiveVMValidators: []*LiveVMValidator{
+				FileHasContentsValidator("/etc/motd", "foobar"),
+			},
+		},
+	})
+}
+
+func Test_azurelinuxv2MessageOfTheDay(t *testing.T) {
```

Comment:
I'd opt for moving towards capitals in test names consistently, `Test_AzureLinuxV2MessageOfTheDay`